tools/proxy/proxy.js

```diff
@@ -52,6 +52,10 @@ const processes = {
       shell: true,
     },
   ),
+  connect: spawnIfNotSpecified("noConnect", "docker-compose run --rm webknossos-connect", [], {
+    cwd: ROOT,
+    shell: true,
+  }),
```

Comment:
@philippotto If I remember correctly, you start the different parts manually sometimes. Does this break anything for you? (sbt run won’t start webknossos-connect)

Reply:
That should be ok for me :)